src/main/resources/application.conf

```diff
@@ -24,4 +24,15 @@ akka {
     seed-nodes = [${?VALUE}]
     auto-down-unreachable-after = 10s
   }
+}
+
+zookeeper {
+//  connection = ${?ZOOKEEPER_CONNECTION} //"master.mesos:2181"
```

Comment:
No comments related with mesos in the configuration file. KH do not need mesos to run and could be confusing for the user